### ISSUE TITLE
Multi-column view

### ISSUE THREAD

**Issue by McKristoffer** [elk-zone/elk#1404]
Clear and concise description of the problem

The "advanced webview" in the original client allowed to setup multiple columns, each with a different timeline/list (home, local, federated, etc.).

Suggestion

Add a similar type of optional view mode that would allow showing different timelines at the same time - obviously not everyone prefers to have it setup this way, hence why it should be enabled in the options menu first.

**Comment by tgarrison3** [comment]
Fedilab handles lists very nicely by allowing you to "mute" that an account from your home timeline when you add them to a list.  Definitely worth looking at what they've done.

**Comment by ZMYaro** [comment]
May I ask why you think this needs to be built into Elk rather than an idea for a separate client?  Even TweetDeck is a separate client from the regular Twitter client.

**Comment by boehs** [comment]
> May I ask why you think this needs to be built into Elk rather than an idea for a separate client? Even TweetDeck is a separate client from the regular Twitter client.

Why not? There's not an abundent amount of implementation complexity, even lists use the same underlying timeline code. Additionally, I have personally recieved multiple requests for this feature inside elk, and implementing it achieves [parity](https://github.com/elk-zone/elk/issues?q=is%3Aopen+label%3A%22c%3A+parity%22+sort%3Aupdated-desc) with the official client

**Comment by NetwarSystem** [comment]
I found this issue because I showed up with the intent to ask for a Tweetdeck like multiple columns setup.

The Mastondon web client offers columns in advanced mode, but only three of them and they are defined by the system, not the user.

I have a large 4k display and it will comfortably display six columns and I've defined fifteen separate decks. I follow no one with this setup, each column is a query on a given name filtered to only show verified accounts.

I like Elk, it feels really light and fast, and it does a sensible job with conversations vs. mentions. As a minimum I would want to be able to define columns so I can see what various lists are saying at a glance. Then for the ultimate win, let me replicate the entire M x N grid of "decks" and columns.

If it helps to visualize, what I'm doing with all those decks is grouping names into sets of six that are related - Vladimir Putin, Valery Gerasimov, Sergei Shoigu, etc. Then another page has Volodomyr Zelenskyy, Ruslan Stefanchuk, etc. The Fediverse is not yet busy enough to need something like this, but given the steady growth and the obvious distress at Twitter, it's only a matter of time before it would move from being a want to an urgent need.

**Comment by kimusan** [comment]
I would love to see this as well. It is one of the things I miss when I moved from Sengi client to Elk. With lists in separate columns, it really makes Mastodon a very strong tool.

**Comment by kimusan** [comment]
> The Fediverse is not yet busy enough to need something like this, but given the steady growth and the obvious distress at Twitter, it's only a matter of time before it would move from being a want to an urgent need.

I do not agree with the "not busy enough" part of this. I am on infosec.exchange instance (among others) and the local (and even worse, the federated) feed is so busy that I can easily miss something and have 2000+ toots waiting when I go to the top to press the "Show XXXX new items" link.

**Comment by Kalytis** [comment]
> May I ask why you think this needs to be built into Elk rather than an idea for a separate client?  Even TweetDeck is a separate client from the regular Twitter client.

Elk clearly is ahead of everybody on multi-account management. Columns view would make it so much more useful for multi-accounts people, orgs, press.

**Comment by uncomfyhalomacro** [comment]
> > The Fediverse is not yet busy enough to need something like this, but given the steady growth and the obvious distress at Twitter, it's only a matter of time before it would move from being a want to an urgent need.
> 
> I do not agree with the "not busy enough" part of this. I am on infosec.exchange instance (among others) and the local (and even worse, the federated) feed is so busy that I can easily miss something and have 2000+ toots waiting when I go to the top to press the "Show XXXX new items" link.

I actually experience this now that I noticed that there are still growing users..

**Comment by Kalytis** [comment]
I love to have a personal and a professionnal account : multi-columns view like Tweetdeck really is amazing to handle that.

**Comment by Moilleadoir** [comment]
I really like the look of Elk, but I wouldn’t switch to it without the option of multi-column display.

**Comment by DJLink** [comment]
+1 for multi-column design :D

**Comment by vengefulpunk** [comment]
i'd switch to Elk if it had multi columns with a clear column option to make it easy to keep track of posts/toots that I have already seen.

**Comment by huggenknubbel** [comment]
since i'm a GoToSocial User and GoToSocial has no web frontend, i would love to have Elk with a Tweetdeck syle multi volumn view
